src/v/config/rjson_serialization.cc

```diff
@@ -80,7 +80,7 @@ void rjson_serialize(
 }
 
 void rjson_serialize(
-  json::Writer<json::StringBuffer>& w, const custom_aggregate& v) {
+  json::Writer<json::StringBuffer>& w, const test_custom_aggregate& v) {
```

Comment:
i didn't intend to include this commit in this PR but it still needs to go in: "core" code should not be including anything from a test directory.